base/all-all/sbardef.lmp

```diff
@@ -0,0 +1,1974 @@
+{
+  "type": "statusbar",
+  "version": "1.1.0",
+  "metadata": null,
+  "data":
+  {
+    "numberfonts":
+    [
+      {
+        "name": "BigRed",
+        "type": 0,
+        "stem": "STT"
+      },
+      {
+        "name": "SmallGrey",
+        "type": 0,
+        "stem": "STG"
+      },
+      {
+        "name": "SmallYellow",
+        "type": 0,
+        "stem": "STYS"
+      }
+    ],
+    "hudfonts":
+    [
+      {
+        "name": "Console",
+        "type": 2,
+        "stem": "STCFN"
+      },
+      {
+        "name": "Digits",
+        "type": 0,
+        "stem": "DIG"
+      }
+    ],
+    "statusbars":
+    [
+      {
+        "height": 200,
```

Comment:
Shoudn't this be `200 - 32` for the status bar hud?

Reply:
It's just `"height": 32` in v1.0.0 lumps. I've  I made it 200 since we have HUD widgets.

Reply:
Okay, I see. It's complicated...